Source/JavaScriptCore/Configurations/JavaScriptCore.xcconfig

```diff
@@ -58,13 +58,18 @@ MODULE_VERIFIER_SUPPORTED_LANGUAGES_NO = objective-c;
 // FIXME: (rdar://74275135) Re-enable support for automatically verifying Objective-C and C when possible.
 MODULE_VERIFIER_SUPPORTED_LANGUAGES_YES = objective-c++ c++;
 
+MODULE_VERIFIER_SUPPORTED_LANGUAGE_STANDARDS = $(MODULE_VERIFIER_SUPPORTED_LANGUAGE_STANDARDS_$(WK_HAS_VALID_JSC_PRIVATE_MODULE));
+MODULE_VERIFIER_SUPPORTED_LANGUAGE_STANDARDS_ = gnu17;
```

Comment:
the gnu17 isnt doing much on these 3 lines since ObjC verification is disabled in JSC for the time being

Reply:
> just gnu++whatever for the YES value

Any ideas how can I find out the default?  Oh, maybe the aforementioned `gnu++20`?

Reply:
the value should be whatever the project uses, which in our case is gnu++23 (I think? or c++23, I forget the difference)

Reply:
It's this from `Source/JavaScriptCore/Configurations/Base.xcconfig`:

```
CLANG_CXX_LANGUAGE_STANDARD = c++2b;
```

Reply:
Welp, not that:

```
Source/JavaScriptCore/JavaScriptCore.xcodeproj: warning: Unsupported module verifier language standard "c++2b" (in target 'JavaScriptCore' from project 'JavaScriptCore')
```

Seems like I need to use just `c++23`.